sdk/examples/transfer-coin.rs

```diff
@@ -25,7 +25,7 @@ static FAUCET_URL: Lazy<Url> = Lazy::new(|| {
         std::env::var("APTOS_FAUCET_URL")
             .as_ref()
             .map(|s| s.as_str())
-            .unwrap_or("https://faucet.devnet.aptoslabs.com"),
+            .unwrap_or("http://localhost:9000"),
```

Comment:
Is this example supposed to be localhost?

Running a local testnet will default to 8081 for this port

Reply:
my mistate. reverted